src/main/scala/com/gd/twitterstreamingtToDatalake/tweetsIngestion.scala

```diff
@@ -0,0 +1,22 @@
+package com.gd.twitterstreamingtToDatalake
+
+import com.gd.twitterstreamingtToDatalake.StreamingTweetsJob.ssc
+import org.apache.spark.streaming.twitter.TwitterUtils
+
+object tweetsIngestion {
```

Comment:
please follow the Scala naming convention guidelines: https://docs.scala-lang.org/style/naming-conventions.html